strapi/strapi
# feat: list history versions in the history page sidebar

### What does it do?

Displays a list of history versions in the history page sidebar

![CleanShot 2024-02-02 at 11 50 59](https://github.com/strapi/strapi/assets/8087692/bcb15775-fb54-43a4-a5a6-066c3420ebcf)


### How to test it?

For any entry, open the history page by clicking on the "history" button in the edit view.

- you should be redirect to the most recent history version (see the id in the URL query params)
- if i18n is enabled, you should only see history versions for the current locale
- clicking on history version cards selects them and updates the query param
- the most recent history version should display `(current)`
- the header of the sidebar should stick to the top, the list of versions should be scrollable
